app/models/dataset/value_set.rb

```diff
@@ -29,7 +29,7 @@ class Dataset::ValueSet < ApplicationRecord
     FoiAttachment
   ].freeze
 
-  validates :resource, :key_set, presence: true
-  validates :resource_type, inclusion: { in: RESOURCE_TYPES }
+  validates :key_set, presence: true
+  validates :resource_type, inclusion: { in: RESOURCE_TYPES }, if: :resource
```

Comment:
If we're not requiring this, what's the reason for keeping the columns?

Reply:
Left in because in Slack you said 
> I’d like to be able to add datasets to info requests outside of projects, so we’ll need to do it at some point

Reply:
Ah, I meant architecturally lets not paint ourselves into a corner where it would be really difficult to do so. Might as well just keep the columns though as there's no harm in them being there.